src/modal/components/ItemModal.jsx

```diff
@@ -13,33 +13,53 @@ const Input = styled.input`
   margin-bottom: 32px;
 `;
 
+// const itemAddedModal = styled.div`
```

Comment:
Remove comments

Reply:
It's part of a feature not yet ready. should i uncomment? remove comments entirely?

Reply:
Which feature? If it's not used in this PR remove entirely